lib/rules/jsx-indent-props.js

```diff
@@ -172,8 +172,8 @@ module.exports = {
      * @param {Number} indent needed indent
      */
     function checkNodesIndent(nodes, indent) {
-      let nestedIndent = indent;
       nodes.forEach((node) => {
+        let nestedIndent = indent;
```

Comment:
this should be reverted; the variable needs to persist across loop iterations

Reply:
Done.